src/utils/githubApi.ts

```diff
@@ -62,11 +64,13 @@ export class GithubRemote {
   public constructor(
     owner: string,
     repo: string,
+    name?: string,
     username?: string,
     apiToken?: string
   ) {
     this.owner = owner;
     this.repo = repo;
+    this.name = name || 'origin';
```

Comment:
We could have `'origin'` be a constant somewhere... but then I don't understand why `prepare.ts` and `publish.ts` doesn't use the same facilities to use git?

Reply:
They both use `utils/githubApi`, or wdym?

Reply:
I enumerated the files incorrectly, sorry.

Second try:

- `src/targets/ghPages.ts` and `src/targets/registry.ts` both use `GithubRemote` from `src/utils/githubApi.ts`
- `src/commands/prepare.ts` doesn't use `GithubRemote`

---

IMHO the constructor here should not read nor otherwise depend on `process.env`. And since there is a use of this class that doesn't provide all arguments to the constructor, we need a default value.

https://github.com/getsentry/craft/blob/21ec582914e08ae8eb1f9b1aa812d28d1d7eac20/src/targets/registry.ts#L36-L39

Reply:
Talked offline with @tonyo, realized this was a mistake since this class is only used for repo clones "owned" by craft in a temp dir.